MapboxNavigation/NavigationMapView.swift

```diff
@@ -625,7 +625,10 @@ open class NavigationMapView: MLNMapView, UIGestureRecognizerDelegate {
                 arrow.minimumZoomLevel = minimumZoomLevel
                 arrow.lineCap = NSExpression(forConstantValue: "butt")
                 arrow.lineJoin = NSExpression(forConstantValue: "round")
-                arrow.lineWidth = NSExpression(format: "mgl_interpolate:withCurveType:parameters:stops:($zoomLevel, 'linear', nil, %@)", MBRouteLineWidthByZoomLevel.multiplied(by: 0.70))
+                arrow.lineWidth = NSExpression(forMLNInterpolating: .zoomLevelVariable,
+                                               curveType: .linear,
+                                               parameters: nil,
+                                               stops: NSExpression(forConstantValue: MLNRouteLineWidthByZoomLevel.multiplied(by: 0.7)))
```

Comment:
Ah ok, this is about https://github.com/maplibre/maplibre-navigation-ios/issues/6?

If it works, I am fine with it, but in general I think we should try to limit PRs to one issue/topic as this is hidden between the SPM changes.

Reply:
Yes this is related to the errors logged to console. I don't feel comfortable releasing an app to our customers when I see errors. I could verify after the changes these errors are not printed anymore and navigation seems to work fine